uvm/servlets/admin/config/network/Interface.js

```diff
@@ -531,7 +532,32 @@ Ext.define('Ung.config.network.Interface', {
                         xtype: 'textfield',
                         vtype: 'ip4Address',
                         emptyText: '[enter IPv4 address]'.t(),
-                        allowBlank: false
+                        allowBlank: false,
+                        validator: function(value) {
+                            // Check for default IP Address    
+                            if(value == '1.2.3.4') return "Deafult address, Modify IPv4 address".t();
```

Comment:
We should not be preventing any IP address, even if default.  Let the duplicate check below catch this condition.

Reply:
Handled default address scenario in `doneEdit` handler function which is invoked on clicking `Done` button on Add/Edit Interface Window.